csharp/autobuilder/Semmle.Autobuild.CSharp/CSharpAutobuilder.cs

```diff
@@ -104,7 +107,9 @@ private CSharpBuildStrategy GetCSharpBuildStrategy()
                 || Options.MsBuildConfiguration != null
                 || Options.MsBuildPlatform != null
                 || Options.MsBuildTarget != null)
+            {
                 return CSharpBuildStrategy.MSBuild;
```

Comment:
Not sure why braces were added here, as the `return` is not a multi-line statement.

Reply:
The condition is multiline, and that's also taken into account. See here: https://github.com/dotnet/roslyn/blob/cf55f3a58e47298426fa971d3bd9d8857c746c65/src/Analyzers/CSharp/Analyzers/AddBraces/CSharpAddBracesDiagnosticAnalyzer.cs#L169-L180